src/Okta.Sdk.UnitTests/LogsClientShould.cs

```diff
@@ -100,7 +99,7 @@ public async Task DeserializeGetLogsJsonResponse()
             log.Target.First().Should().BeOfType<LogTarget>();
             log.Target.First().AlternateId.Should().Be("john-okta@example.com");
             log.Target.First().DisplayName.Should().Be("John Okta");
-            log.Target.First().DetailEntry.GetData().Should().BeEmpty();
+            log.Target.First().DetailEntry.Should().BeNull();
```

Comment:
These changes imply a behavior change somewhere.  Will this cause a major or minor version bump?